doc/source/data.rst

```diff
@@ -82,9 +82,18 @@ in wide format.
 
 .. note::
 
-    When using custom columns for annotations:
-    **pyam** drops any data rows where the 'value' column is 'nan',
-    and it raises an error for 'nan' in any other column.
+    If there are :code:`numpy.nan` in a **pandas.DataFrame**
```

Comment:
@znicholls, I changed this docstring which was originally introduced by you... If you have a minute to take a look, it would be appreciated!